charts/incubator/bookstack/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v2
+appVersion: v21.04.2
+description: A simple, self-hosted, easy-to-use platform for organising and storing information.
+name: bookstack
+version: 0.1.0
```

Comment:
You can move this to stable and make the version 1.0.0. There isn't much testing to be done in this chart. It's pretty cut and dry